torchtune/data/_messages.py

```diff
@@ -175,30 +175,54 @@ def __init__(
     ):
         self.train_on_input = train_on_input
```

Comment:
I have this slight nagging feeling that this transform is over-generalized and it wouldn't be harmful to define another transform which should properly document that the column map can include image, and to also include validation logic, and without needing to check for multimodal.

Fine to leave this as a followup but if there's consensus here I'd like to see an issue.

Reply:
I see this is the same for `ShareGPTToMessages`, so maybe it's fine. We need to properly document MM usage either way.

Reply:
Yeah, I see what you mean. Especially as we add more modalities this may get bloated quick. I was the one who initiated this design in ShareGPTToMessages but I think it's worth reconsidering. I'll put up an issue.